app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_detail.html

```diff
@@ -1,24 +1,76 @@
+                   href="#v-pills-home" role="tab" aria-controls="v-pills-home"
+                   aria-selected="true"><i
+                        class="fas fa-microscope fa-fw"></i>&nbsp;{{ object.title }}
+                </a>
+                <a class="nav-link" id="v-pills-editors-tab" data-toggle="pill"
+                   href="#v-pills-editors" role="tab" aria-controls="v-pills-editors"
+                   aria-selected="false"><i class="fas fa-user fa-fw"></i>&nbsp;Editors
+                </a>
+                <a class="nav-link" id="v-pills-readers-tab" data-toggle="pill"
+                   href="#v-pills-readers" role="tab" aria-controls="v-pills-readers"
+                   aria-selected="false"><i class="fas fa-users fa-fw"></i>&nbsp;Readers
+                </a>
+                <a class="nav-link" id="v-pills-cases-tab" data-toggle="pill"
+                   href="#v-pills-cases" role="tab" aria-controls="v-pills-cases"
+                   aria-selected="false"><i class="fas fa-image fa-fw"></i>&nbsp;Cases&nbsp;<span
+                        class="badge badge-pill badge-secondary align-middle">{{ object.images.all.count }}</span>
+                </a>
+                <a class="nav-link" id="v-pills-questions-tab" data-toggle="pill"
+                   href="#v-pills-questions" role="tab"
+                   aria-controls="v-pills-questions"
+                   aria-selected="false"><i class="fas fa-question fa-fw"></i>&nbsp;Questions&nbsp;<span
+                        class="badge badge-pill badge-secondary align-middle">{{ object.questions.all.count }}</span>
+                </a>
+                <a class="nav-link" id="v-pills-ground-truth-tab" data-toggle="pill"
+                   href="#v-pills-ground-truth" role="tab"
+                   aria-controls="v-pills-ground-truth"
+                   aria-selected="false"><i class="fas fa-check fa-fw"></i>&nbsp;Ground
+                    Truth
+                </a>
+                <a class="nav-link"
+                   href="{% url 'reader-studies:leaderboard' slug=object.slug %}"><i
+                        class="fas fa-trophy fa-fw"></i>&nbsp;Leaderboard
+                </a>
+                <a class="nav-link"
+                   href="{% url 'reader-studies:statistics' slug=object.slug %}"><i
+                        class="fas fa-chart-bar fa-fw"></i>&nbsp;Statistics
+                </a>
+                <a class="nav-link" id="v-pills-export-tab" data-toggle="pill"
+                   href="#v-pills-export" role="tab" aria-controls="v-pills-export"
+                   aria-selected="false"><i class="fas fa-download fa-fw"></i>&nbsp;Export
+                </a>
+            </div>
+        </div>
+    {% endif %}
+{% endblock %}
 
-    <h1>Reader Study {{ object.title }}</h1>
+{% block content %}
+    {% get_obj_perms request.user for object as "readerstudy_perms" %}
 
     {% if not object.is_valid %}
```

Comment:
Something I just noticed and might as well be part of this PR: this message also shows for readers. We should also check if the user is an editor.

Reply:
Updated